openshift/file-integrity-operator
# install: Switch to using bundles

These patches are equivalent of similar patches that use bundles instead
of manifests. I haven't pushed the images to FI quay.io repo, but it's easy
to test them out:

- make image
- tag and push the operator image and the bundle image, e.g. I had to call:
    - podman tag 85123f81161c quay.io/jhrozek/file-integrity-operator-bundle:latest
    - podman push quay.io/jhrozek/file-integrity-operator-bundle:latest
    - podman tag e840d82fb613 quay.io/jhrozek/file-integrity-operator:latest
    - podman push quay.io/jhrozek/file-integrity-operator:latest 
- build and push the index:
    - opm index add -b quay.io/jhrozek/file-integrity-operator-bundle:latest -t quay.io/jhrozek/file-integrity-operator-index:latest -c podman
    - podman push quay.io/jhrozek/file-integrity-operator-index:latest
- modify the catalog source to point to your index you just pushed
- follow the instructions in the latest commit to install the operator

If these patches are pushed, we should probably also amend the docs.
